src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
Sorry reading my earlier comment I think there is some miss-understanding. I wasn't implying we stop using CMake I was just implying that we should use CMake the same way on windows and linux/osx. Honestly I'm not entirely sure how this works on linux/osx (I should dig into this more) but all I see is that we have a bunch of extra scripts specific to windows which I would like to eliminate if possible.